napari/layers/points/points.py

```diff
@@ -619,7 +619,7 @@ def edge_color_cycle(
 
     @property
     def edge_colormap(self):
-        """(str, vispy.color.Colormap) colormap to be applied to a property to set edge_color"""
+        """(str, vispy.color.Colormap): colormap to be applied to a property to set edge_color"""
```

Comment:
@kevinyamauchi I haven't seen any docstring convention documents deal with properties specifically, but I would follow the convention for methods, I think:

```python
"""Return the colormap to be applied to a property to get the edge color.

Returns
-------
colormap_name : str
    The name of the current colormap.
colormap : vispy.color.Colormap
    The actual colormap object.
"""

Reply:
I looked and couldn't find any either. I will update them to follow the convention for methods.

Reply:
To avoid departing from the style of the other layers, I changed the docstrings that didn't work as one-liners and left the rest as-is.